packages/core/src/components/field-label/field-label.scss

```
@@ -0,0 +1,19 @@
+@use '../../helpers';
+
+@mixin FieldLabel() {
+  .ods-field-label {
+    @include helpers.font('300-regular');
+
+    color: helpers.color('content-main');
+    display: flex;
+    flex-flow: column wrap;
+    padding-top: helpers.space(0.5);
+
+    + input,
+    + textarea,
+    > input,
+    > textarea {
```

Shouldn't these be our classes?

```suggestion
    + .ods-input,
    + .ods-textarea,
    > .ods-input,
    > .ods-textarea {
```

---

LOL, yes...